courses/machine_learning/deepdive/08_image/mnist_linear.ipynb

```diff
@@ -9,9 +9,7 @@
    "source": [
```

Comment:
<p>I'd split the two lines</p><p> </p><p>print(x_train[0].shape)</p><p>tf.print(tf.size(tf.unique(y_train).y))</p><p> </p><p>into their own cell and prefix it with a markdown cell explaining what we are doing (especially for the last line)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Yeah, good call. That's much cleaner.</p>